test/Microsoft.ML.StaticPipelineTesting/Microsoft.ML.StaticPipelineTesting.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <DefineConstants>CORECLR</DefineConstants>
+    <OldToolsVersion>2.0</OldToolsVersion>
```

Comment:
What's this for?

Reply:
Something added by VS I guess. I'll remove it and see if VS complains.

---
In reply to: [214523228](https://github.com/dotnet/machinelearning/pull/778#discussion_r214523228) [](ancestors = 214523228)